### ISSUE TITLE
Create Lastpass process

### ISSUE THREAD

**Issue by ahmadabugosh** [GeneralMagicio/gm-planning#118]

**Comment by Zeptimus** [comment]
I don't think we need decentralized management of credentials correct me if I'm wrong. In that way of thinking I would have 2 admins who have access to everything but do not have to be particularly working on any platform. Their role would be to manage the access, audit who has access to what, and coordinate with "platform admins"

Platform admins are the managers of the platform having a work role in the platform. Those are responsible for adding new people to the platform and deciding wich privilege should the user have (admins are there only to coordinate, keep track of who has access to what and be a point of contact if needed)

**Comment by ahmadabugosh** [comment]
Thanks @Zeptimus ! That's pretty much what we did with @mhmdksh and I, but thanks for the push! 

Speaking of which @mhmdksh can you take over this issue so we can close it? Making sure we have all the passwords on the shared sheet and in Lastpass? (I'll dm you on Discord too)
